package.json

```diff
@@ -38,7 +39,7 @@
     "typescript": "4.0.2"
   },
   "scripts": {
-    "build": "microbundle",
+    "build": "npm run prettier && microbundle && tsc && node build/scss.js",
```

Comment:
Using `ts-node` would help eliminate managing the generated files. Could you test with the following;


```
 "build": "npm run prettier && microbundle && npx ts-node build/scss.ts",
```

?

Reply:
Nitpick: In the future, we should split up the formatting from the build step.

Reply:
I can't seem to get the ts-node part working. Could we do that part later?

Reply:
SGTM!